cw/cw305/capture.py

```diff
@@ -152,6 +175,102 @@ def capture_aes_random(ot, ktp):
         yield ret
 
 
+def capture_aes_random_batch(ot, ktp, capture_cfg, scope_type):
+    """A generator for capturing AES traces in batch mode.
+    Fixed key, Random texts.
+
+    Args:
+      ot: Initialized OpenTitan target.
+      ktp: Key and plaintext generator.
+      capture_cfg: Capture configuration.
+      scope_type: cw or waverunner as a scope for batch capture.
+    """
+    # Seed host's PRNG.
+    # TODO: Replace this with a dedicated PRNG to avoid other packages breaking our code.
+    random.seed(capture_cfg["batch_prng_seed"])
+    # Set the target's key
+    key = ktp.next_key()
+    tqdm.write(f'Using key: {binascii.b2a_hex(bytes(key))}')
+    cipher = AES.new(bytes(key), AES.MODE_ECB)
```

Comment:
```suggestion
```

Reply:
Should we use scared function instead?

Reply:
No, let's not do that now. We can file a separate PR to use the same cipher everywhere. This PR should just be about merging the batch script into the regular one.